src/components/referrals/ReferralsComponentLabel.vue

```diff
@@ -1,8 +1,19 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
-        <h5 class="documentBox__title">{{ name }}</h5>
-        <p class="documentBox__data">{{ date }}</p>
+        <h5
+            :aria-label="'Imię i nazwisko lekarza' + name"
+            class="documentBox__title"
+            tabindex="0">
+            {{ name }}
+        </h5>
+
+        <p
+            :aria-label="'Data wystawienia skierowania' + date"
```

Comment:
I think that SR reads date without passing date to this aria atrr but i m not sure

Reply:
no, in SR read properly